Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
```

Comment:
Would an alpine image make sense here to keep the container size as small as possible?